pkg/app/app.go

```diff
@@ -15,14 +18,21 @@ type AddCommandContext struct {
 	NoColor    bool
 	Dir        bool
 	Concurrent int
+	Filters    []string
 }
 
 //NewAddContextFromCli is a constructor which creates the context
 func NewAddContextFromCli(c *cli.Context) AddCommandContext {
```

Comment:
Do we want to create dedicated struct for TextContext now, rather than backpacking off of add?

Reply:
Can you explain it a little bit more? I don't know where we are using a `TextContext`

Reply:
Oh wow.. Apologies. What I meant to say here is.. Should we create a new struct for `TestCommandContext`? Right now we just backpack off of `AddCommandContext`, as we continue to add features these will continues to differentiates themselves. Doe that help?

Reply:
Ahhh, that makes sense ;)
Renamed it :+1: